src/chrome/content/rules/CafePress.xml

```diff
@@ -1,41 +1,33 @@
-
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://server.iad.liveperson.net/ => http://server.iad.liveperson.net/: (35, 'Unknown SSL protocol error in connection to solutions.liveperson.com:443 ')
 	!functional:
 		- blog.cafepress.com		(timeout)
 		- images7?.cafepress.com	(reset; redirects to i2.cpcache.com)
 		- i[12].cpcache.com		(503, akamai; !at i[12].cafepress.com, content.cafepress.com, www.cafepress.com)
 -->
-<ruleset name="CafePress (partial)" default_off='failed ruleset test'>
-
-	<target host="cafepress.com"/>
-	<target host="*.cafepress.com"/>
-	<target host="cafepress.co.uk"/>
-	<target host="*.cafepress.co.uk"/>
-	<target host="*.cpcache.com"/>
-	<target host="server.iad.liveperson.net"/>
-
-	<rule from="^http://(?:www\.)?cafepress\.co(m|\.uk)/content/"
-		to="https://www.cafepress.co$1/content/"/>
-
-	<rule from="^http://(content|members)\.cafepress\.co(m|\.uk)/"
-		to="https://$1.cafepress.co$2/"/>
-
-	<rule from="^http://shop\.cafepress\.com/$"
-		to="https://shop.cafepress.com/"/>
-
-	<!--	cert !match help	-->
-	<rule from="^http://help\.cafepress\.com/hc/"
-		to="https://server.iad.liveperson.net/hc/"/>
+<ruleset name="CafePress (partial)">
+	<target host="cafepress.com" />
+	<target host="content.cafepress.com" />
+	<target host="members.cafepress.com" />
+	<target host="shop.cafepress.com" />
+	
+	<target host="cafepress.co.uk" />
+	<target host="members.cafepress.co.uk" />
+	
+	<target host="content.cpcache.com" />
+	<target host="content0.cpcache.com" />
+	<target host="content1.cpcache.com" />
+	<target host="content2.cpcache.com" />
+	<target host="content3.cpcache.com" />
+	<target host="content4.cpcache.com" />
+	<target host="content5.cpcache.com" />
+	<target host="content6.cpcache.com" />
+	<target host="content7.cpcache.com" />
+	<target host="content8.cpcache.com" />
+	<target host="content9.cpcache.com" />
 
 	<!--	Akamai bucket.  Finding it would ideal.	-->
 	<rule from="^http://content\d?\.cpcache\.com/"
 		to="https://content.cafepress.com/"/>
```

Comment:
Please add a valid test url for this

Reply:
@J0WI Already added
```xml
	<target host="content0.cpcache.com" />
	<target host="content1.cpcache.com" />
	<target host="content2.cpcache.com" />
	<target host="content3.cpcache.com" />
	<target host="content4.cpcache.com" />
	<target host="content5.cpcache.com" />
	<target host="content6.cpcache.com" />
	<target host="content7.cpcache.com" />
	<target host="content8.cpcache.com" />
	<target host="content9.cpcache.com" />
```

Reply:
I get a 403 there. I would prefer a valid resource for redirects to a different host.